resources/views/cart/coupon.blade.php

```diff
@@ -1,34 +1,47 @@
-<div class="bg-white rounded-lg border sm:w-80 mb-3">
-    <div class="p-3">
-        <h3 class="text-lg leading-6 font-medium text-gray-900">
-            @lang('Apply coupon code')
-        </h3>
-        <coupon v-slot="{ cart, removeCoupon, couponCode, inputEvents, applyCoupon }">
-            <div>
-                <form class="mt-5 flex" @submit.prevent="applyCoupon">
-                    <x-rapidez::input
-                        :label="false"
-                        name="couponCode"
-                        placeholder="Coupon code"
-                        v-on="inputEvents"
-                        v-bind:value="couponCode"
-                        v-bind:disabled="$root.loading"
-                    />
-
-                    <x-rapidez::button type="submit" class="ml-3 sm:text-sm">
-                        @lang('Apply')
-                    </x-rapidez::button>
-                </form>
-
-                <div class="relative rounded-md" v-if="cart.discount_name && cart.discount_amount < 0">
-                    <div class="flex items-center">
-                        <button v-on:click="removeCoupon">
-                            <x-heroicon-s-x class="h-4 w-4 text-black-400"/>
-                        </button>
-                        @lang('Discount'): @{{ cart.discount_name }}
+<toggler>
```

Comment:
is it me, or can this file be minimalized, allot of <div's and elements, just for a small form with a button? seems a bit over the top, can you skim this down, sure it has te be readable but it looks just so much for a simple form?